snapcraft.yaml

```diff
@@ -190,11 +193,22 @@ parts:
         echo "Preparing istio"
         cp $KUBE_SNAP_BINS/istioctl .
         cp -r $KUBE_SNAP_BINS/istio-yaml ./actions/istio
+        # Add bash completion for istioctl. This is located here to make sure it is only added when istio is added.
```

Comment:
I believe we also need a `touch istioctl.bash` for non-amd64 architectures here   
https://github.com/ubuntu/microk8s/pull/254/files#diff-184032a532406b07009403e26f4fc62fR188 . In non-amd64 architectures the `microk8s.istioctl` command still exists but the wrapper tells you that istio is not available for the current architecture.